iodata/formats/qchem.py

```diff
@@ -0,0 +1,251 @@
+            break
+    next(lit)
+    for line in lit:
+        if not line.strip().startswith('i: 0 AtomOff'):
+            polar_matrix.append(line.strip().split()[1:])
+        else:
+            break
+    polar_matrix = np.array(polar_matrix, dtype=np.float)
+
+    # Get Hessian of the SCF energy
+    athessian = np.zeros((3 * num_atoms, 3 * num_atoms), float)
+    if lit_hess is None:
+        for line in lit:
+            if line.strip().startswith(
+                    'Hessian of the SCF Energy') or \
+                    line.strip().startswith('Final Hessian'):
+                break
+        nb = int(np.ceil(num_atoms * 3 / 6))
+        for i in range(nb):
+            next(lit)
+            row = 0
+            for line in lit:
+                words = line.split()
+                # / angstrom**2
+                athessian[row, 6 * i:6 * (i + 1)] = np.array(
+                    sum([[float(word)] for word in words[1:]], []))
+                row += 1
+                if row >= 3 * num_atoms:
+                    break
+    # or get Hessian from other file
+    else:
+        row = 0
+        col = 0
+        for line in lit_hess:
+            athessian[row, col] = float(line.split()[0]) * 1000 * calorie / avogadro / angstrom ** 2
+            col += 1
+            if col >= 3 * num_atoms:
+                row += 1
+                col = row
+        for i in range(len(lit_hess)):
+            for j in range(0, i):
+                athessian[i, j] = athessian[j, i]
+
+    # get masses in atomic units
+    masses = np.zeros(num_atoms, np.float)
+    for line in lit:
+        # Vibrational energy
+        if line.strip().startswith('Zero point vibrational energy'):
+            vib_energy = np.array(line.strip().split()[-2],
+                                  dtype=np.float)
+            break
+    next(lit)
+    count = 0
+    for line in lit:
+        masses[count] = float(line.split()[-1]) * amu
+        count += 1
+        if count >= num_atoms:
+            break
+
+    # get symmetry number
+    for line in lit:
+        if line.strip().startswith('Rotational Symmetry Number is'):
+            num_sym = np.array(line.split()[-1], dtype=np.int)
+        # Enthalpy in kcal/mol
+        if line.strip().startswith('Total Enthalpy'):
+            enthalpy = np.array(line.strip().split()[-2], dtype=np.float)
+        # Entropy in cal/mol.K
+        if line.strip().startswith('Total Entropy'):
+            entropy = np.array(line.strip().split()[-2], dtype=np.float)
+            break
+
+    return title, runtype, basis, exchange, atnames, atnums, num_atoms, num_sym, \
+        masses, atcoords, polar_matrix, athessian, nucl_repul_energy, \
+        num_alpha_electron, num_beta_electron, mol_charges, \
+        mulliken_charges, energy, alpha_mo_energy, beta_mo_energy, \
+        vib_energy, enthalpy, entropy
+
+
+def load_one(lit: LineIterator, lit_hess: LineIterator = None) -> dict:
```

Comment:
All `load_one` functions in `IOData.formats` should follow the same API, i.e. just take one `lit` argument. In this case, I would suggest to make a second module `iodata.formats.qchem_hessian` for the code that can read the QChem Hessian from a separate file.